app/views/shared/_recovery_code_input.html.slim

```diff
@@ -0,0 +1,10 @@
+input(
+  aria-label="#{t('forms.personal_key.confirmation_label')}"
+  maxlength="#{PersonalKeyFormatter.code_length}"
```

Comment:
Are we consistent elsewhere that we are using the PersonalKeyFormatter for managing lengths and regexes for recovery codes? If not, I'd be more comfortable copying PersonalKeyFormatter into RecoverCodeFormatter so we don't have to worry about divergence later.

Reply:
This is yet another partial that is not used.